sdk/formrecognizer/Azure.AI.FormRecognizer/src/FormPage.cs

```diff
@@ -85,11 +85,12 @@ internal FormPage(int pageNumber, float width, float height, float textAngle, Le
         public LengthUnit Unit { get; }
 
         /// <summary>
-        /// When 'IncludeFieldElements' is set to <c>true</c>, a list of recognized lines of text.
-        /// An empty list otherwise. For calls to recognize content, this list is always populated. The maximum number of
-        /// lines returned is 300 per page. The lines are sorted top to bottom, left to right, although in certain cases
-        /// proximity is treated with higher priority. As the sorting order depends on the detected text, it may change across
-        /// images and OCR version updates. Thus, business logic should be built upon the actual line location instead of order.
+        /// When 'IncludeFieldElements' is set to <c>true</c>, a list of recognized lines of text. An empty list otherwise.
+        /// The maximum number of lines returned is 300 per page. The lines are sorted top to bottom, left to right, although
+        /// in certain cases proximity is treated with higher priority. As the sorting order depends on the detected text, it
+        /// may change across images and OCR version updates. Thus, business logic should be built upon the actual line location
+        /// instead of order. For calls to recognize content, this list is always populated and the sorting algorithm for the
+        /// returned lines can be changed with the option <see cref="RecognizeContentOptions.ReadingOrder"/>.
         /// </summary>
         public IReadOnlyList<FormLine> Lines { get; }
```

Comment:
to make sure I understood, there is no default ReadingOrder applied, right?

Reply:
We're not setting anything to be the default on the client side, but if we don't send anything, the service will use the `Basic` algorithm.